python/doc/examples/functional_modeling/quick_start_functions.ipynb

```diff
@@ -0,0 +1,585 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<p>[6,-5]</p>"
+      ],
+      "text/plain": [
+       "class=Point name=Unnamed dimension=2 values=[6,-5]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "myfunction([1.,2.,3.])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However, the newly created `myfunction` has services that the basic Python function did not have. For example, we can create a `CompositeRandomVector` on top of it, by associating it to the input random vector."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "outputVect =  ot.CompositeRandomVector(myfunction, inputRandomVector)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following example, we estimate the output mean based on a simple Monte-Carlo experiment using 10000 function evaluations. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "montecarlosize = 10000\n",
+    "outputSample = outputVect.getSample(montecarlosize)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[-0.0166778,-0.0123527]\n",
+      "[1.73359,1.39888]\n"
+     ]
+    }
+   ],
+   "source": [
+    "empiricalMean = outputSample.computeMean()\n",
+    "print(empiricalMean)\n",
+    "empiricalSd = outputSample.computeStandardDeviationPerComponent()\n",
+    "print(empiricalSd)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What types for x and for y ?\n",
+    "\n",
+    "Not all types are possible for the inputs and outputs of the `mySimulator` function. The following table present some of the available types. All in all, any type which can be converted to or from a \"vector\" can be managed by the `PythonFunction` class.\n",
+    "\n",
+    "| Type | Input X | Output Y |\n",
+    "|---|---|---|\n",
+    "| `list` (Python) | - | ✓ |\n",
+    "| `tuple` (Python) | - | ✓ |\n",
+    "| `array` (NumPy) | - | ✓ |\n",
+    "| `Point` (OpenTURNS) | ✓ | ✓ |\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The vectorized Python function\n",
+    "\n",
+    "The `PythonFunction` class has a `func_sample` option which vectorizes the computation so that all the output values in the sample are computed from a single function call, without any `for` loop. To make this possible, the input and output is then a `Sample` instead of a `Point`. In general, this boosts performance (but not always).\n"
```

Comment:
```suggestion
    "The `PythonFunction` class has a `func_sample` option which vectorizes the computation so that all the output values in the sample are computed from a single function call, without any `for` loop. To make this possible, the input and output is then a `Sample` instead of a `Point`. This often boosts performance (but not always).\n"
```

Reply:
Fixed.